x/compute/client/utils/utils.go

```diff
@@ -252,7 +252,7 @@ var re = regexp.MustCompile("encrypted: (.+?):")
 func (ctx WASMContext) DecryptError(errString string, msgType string, nonce []byte) (json.RawMessage, error) {
 	regexMatch := re.FindStringSubmatch(errString)
 	if len(regexMatch) != 2 {
-		return nil, fmt.Errorf("Got an error finding base64 of the error: regexMatch '%v' should have a length of 2", regexMatch)
+		return nil, fmt.Errorf("Got an error finding base64 of the error: regexMatch '%v' should have a length of 2. error: %v", regexMatch, errString)
 	}
```

Comment:
This is the code that prints an error when the query fails. I changed this in order to be able to see the TCS message in the CLI in my testing.